src/auto-evo/EvolutionaryTreeNode.cs

```diff
@@ -63,7 +79,7 @@ public override void _Ready()
 
     private void UpdateTexture()
     {
-        if (lastGeneration)
```

Comment:
What's this? The red mark was intended to convey that it is the last node of a species. Why do you just modify its usage? And, I think *red* flag is not very friendly, you can request blue textures.

Reply:
I think the highlighting should also have a custom colour different from the extinct or normal.

Reply:
Indeed, the idea was to eventually revert back to it. But I had a question to tackle: what should we do for a species that is both flagged and last generation?

Reply:
Flag is of higher priority of course.

Reply:
It was actually suggested by Nick to use inner and outer rings of the button to mark both.

Reply:
I think it's not worth the trouble.

Reply:
Second thought: we can set the modulate of the unwanted (filtered out) nodes half-transparent.